src/ThirdParty/ThirdParty.errors.ts

```diff
@@ -0,0 +1,5 @@
+export class NonExistentThirdPartyError extends Error {
+  constructor(public id: string) {
```

Comment:
Just out of curiosity, do we need/want to use the `id` passed in the constructor?